docs/pages/setup/reference/cli.mdx

```diff
@@ -685,6 +685,54 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh exec
+
+Run provided command, by default using a shell, with Teleport-specific environment
+variables set. Can be used to create combined commands.
+
+The command can be used to run external executables, defaulting to a shell. To
+find a usable shell, `tsh exec` will try in order: `$SHELL` env variable, `sh`
+, `bash`. The flag `--shell` allows for overriding of the executable to run,
+while all arguments passed to command are fed as-is.
+
+The command will be executed under modified environment, enriched with the following variables:
+
+| Name          | Value                                                |
+|---------------|------------------------------------------------------|
+| `TSH_COMMAND` | The program being executed.                          |
```

Comment:
Is this the full path to the executable? Does it include the arguments?

Reply:
Just executable, without arguments.